core/src/templates/timeline-page.js

```diff
@@ -24,17 +25,71 @@ const StyledDesc = styled(Typography)`
   }
 `
 
-function EventFooter({ event }) {
-  if (event.extra.buttonText && event.extra.link) {
-    return (
-      <ButtonGroup fullWidth style={{ padding: 16 }}>
-        <Button variant="outlined" color="inherit" onClick={() => event.onClick()}>
-          {event.extra.buttonText}
-        </Button>
-      </ButtonGroup>
-    )
+const FullwidthButton = styled(Button)`
+  && {
+    width: 100%;
   }
-  return null
+`
+
+function EventCreditListSubheader({ title }) {
+  return (
+    <ListSubheader component="div" style={{lineHeight: "30px", textAlign: "left"}}>
+      { title }
+    </ListSubheader>
+  )
+}
+
+function EventCreditList({ title, credits }) {
+  const validCredits = credits.filter(c => !!c.name)
+  return (
+    <List subheader={<EventCreditListSubheader title={title}/>} dense>
+      {
+        validCredits.map((c, idx) => {
+          if (c.link == null || c.link.length == 0) {
```

Comment:
since `c.link` should be string, so we can
```suggestion
          if (!c.link) {
```

Reply:
Fixed. Please check